doc/sphinx/language/core/basic.rst

```diff
@@ -0,0 +1,509 @@
+=============================
+Basic notions and conventions
```

Comment:
Good enough for now.  Maybe later on, as the document evolves, a better title will be obvious.